blackmouse572/quizler-fe
# Feat/login page

## Main feat:
- Add login page
- Add login form
- Add login action that will fetch a fake api and then store `token` into cookies 
- Add helper that attach/remove token to cookies

## Test suggestion
- Click **Sign In** Button at main page or go to http://localhost:3000/login
- Fill in correct credentials: 
```json
{
  "email": "john@mail.com",
  "password": "changeme"
}
```
- Press **Sign In** Button
Expected Result: 
- Success toast pop up and then automatically navigate back to landing page
- Landing page should display message "If you see this. You are authenticated!"

## Preview:
![Login](https://github.com/blackmouse572/quizler-fe/assets/77144199/6cd18d14-c72b-431e-be06-f1f28b1aed2e)
![Logining in](https://github.com/blackmouse572/quizler-fe/assets/77144199/7b3db636-4631-4aef-b655-cfdcf66f8490)
![Login main page](https://github.com/blackmouse572/quizler-fe/assets/77144199/e4664125-5436-4611-97bf-ec5bc64cf886)
